libp2p/CHANGELOG.md

```diff
@@ -6,9 +6,17 @@
 - Raise MSRV to 1.73.
   See [PR 5266](https://github.com/libp2p/rust-libp2p/pull/5266).
 
+- Implement refactored `Transport`.
+  See [PR 4568].
+
+- Move `address_translation` from `libp2p-core` to `libp2p-swarm` and `libp2p-identify`.
```

Comment:
IMO, this needs to be clearer that you now have to depend on `libp2p-identify` to get address translation behaviour, i.e. discovery of external address (candidates) based on connecting to other peers.